api-definitions/db_positions/api.yaml

```diff
@@ -290,6 +297,14 @@ definitions:
         type: string
         minLength: 1
         example: psp1234567890
+      psp_fiscal_code:
+        type: string
+        minLength: 1
+        example: PSP fiscal code
+      psp_vat_number:
+        type: string
+        minLength: 1
+        example: PSP VAT number
```

Comment:
ditto

Reply:
We changed the field to description, also we put no example as there was no available example value in the DB.